plugins/orchestrator/src/components/WorkflowsTable.tsx

```diff
@@ -104,18 +106,21 @@ export const WorkflowsTable = ({ items }: WorkflowsTableProps) => {
         title: 'Last run status',
         field: 'lastRunStatus',
         render: rowData =>
-          rowData.lastRunStatus !== VALUE_UNAVAILABLE ? (
-            <WorkflowInstanceStatusIndicator
-              status={rowData.lastRunStatus as ProcessInstanceStateValues}
-            />
+          rowData.lastRunStatus !== VALUE_UNAVAILABLE &&
+          rowData.lastRunId !== VALUE_UNAVAILABLE ? (
+            <Link to={workflowInstanceLink({ instanceId: rowData.lastRunId })}>
```

Comment:
The link should also appear in the workflow details page
move the Link to be under WorkflowInstanceStatusIndicator so it'll be reused in both places

Reply:
Done